### PR TITLE
Issues/412 build fido2 tests

### PR DESCRIPTION
**Pull request by zcgandcomp**

### CODE REVIEW

File: .github/workflows/maven-deploy-test-server.yml [hunk 1]
```diff
@@ -10,13 +10,13 @@ on:
 jobs:
   maven-deploy-manual:
     if: ${{ github.event_name == 'workflow_dispatch' }}
-    name: Manual deploy
-    uses: wultra/wultra-infrastructure/.github/workflows/maven-deploy.yml@develop
+    name: Manual deploy test-server
+    uses: wultra/wultra-infrastructure/.github/workflows/maven-deploy-test-server.yml@develop
```
Comment (banterCZ) [inline .github/workflows/maven-deploy-test-server.yml:14]: Are you sure that this shared action exists?

File: .github/workflows/maven-deploy-fido2-tests.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Deploy FIDO2 Tests with Maven
```
Comment (banterCZ) [inline .github/workflows/maven-deploy-fido2-tests.yml:1]: We build two WARs of PAS by a single action. But why not, we may split it here. It is probably better to have such granularity.

File: .github/workflows/maven-deploy-test-server.yml (hunk 1 above)
Comment (zcgandcomp) [reply .github/workflows/maven-deploy-test-server.yml:14]: Oops, renaming the file in the IDE has some consequences.

File: .github/workflows/maven-deploy-fido2-tests.yml (hunk 2 above)
Comment (zcgandcomp) [reply .github/workflows/maven-deploy-fido2-tests.yml:1]: The problem are other modules so we need to call only build of test server and fido2 tests.
And we cannot do it in one action as the resulting artifacts will clash on name.

**Pull request merged**